client/tx/search.go

```diff
@@ -98,6 +98,7 @@ $ gaiacli query txs --tags '<tag1>:<value1>&<tag2>:<value2>' --page 1 --limit 30
 	cmd.Flags().String(flagTags, "", "tag:value list of tags that must match")
 	cmd.Flags().Int32(flagPage, defaultPage, "Query a specific page of paginated results")
 	cmd.Flags().Int32(flagLimit, defaultLimit, "Query number of transactions results per page returned")
+	cmd.MarkFlagRequired(flagTags)
```

Comment:
If a flag is required, please evaluate the possibility to turn into an argument

Reply:
I was trying to avoid too many changes to the CLI. I found a BUNCH of places where we can do that and will push a followup PR with some changes like that.